### PR TITLE
Link executant to facility

### PR DESCRIPTION
**Pull request by kostya9**

### CODE REVIEW

File: backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+BEGIN;
```
Comment (k-gregory) [inline backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql]: By default, Flyway always wraps the execution of an entire migration within a single transaction.

File: backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+ALTER TABLE executant
+    ADD COLUMN facility_id bigint references facility (id);
+
+UPDATE executant SET facility_id=1 WHERE facility_id IS NULL;
```
Comment (k-gregory) [inline backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql]: alter table executant add column facility_id bigint references facility(id) not null default 1;
alter table executant alter column facility_id drop default;

**Review by k-gregory — CHANGES_REQUESTED** [review]

File: backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql (hunk 1 above)
Comment (kostya9) [reply backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql]: Cool!

File: backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql (hunk 2 above)
Comment (kostya9) [reply backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql]: Wanted to explicitly state the assignment of facilities, but my way doesn't account of entries being inserted between `UPDATE` AND `SET NOT NULL` statements.

Will use your proposed approach

**Pull request merged**